cookbooks/cdo-apps/test/integration/default/serverspec/ruby_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'serverspec'
```

Comment:
Can you add a comment explaining what is verified by this integration test?  e.g. Does it bring up a full server and verify that it can hit the home page?